content/python/concepts/scope/scope.md

```diff
@@ -0,0 +1,58 @@
+---
+Title: 'Scope'
+Description: 'The variable is only usable inside the region it is created is called as scope in python'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+  - 'Web Development'
+Tags:
+  - 'Lists'
+  - 'Arrays'
+  - 'Objects'
+  - 'Data Structures'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The variable is only usable inside the region it is created is called as **scope** in python. Now this region can be anything a class, method(function), loops, if statements, etc.
```

Comment:
```suggestion
Scope refers to the areas in which variables are accessible. Variables that can be accessed anywhere in a Python are in the global scope. Conversely, variables that are defined within the body of structures like classes and methods exist in the local scope.
```

Docs entries should always have a manual-like tone that is inviting and informative.